[Expo] Execution failed for task ':react-native-libsodium:buildCMakeRelWithDebInfo[arm64-v8a]'

When building the app locally I'm getting the following error:

` [RUN_GRADLEW] FAILURE: Build failed with an exception.
[RUN_GRADLEW] * What went wrong:
[RUN_GRADLEW] Execution failed for task ':react-native-libsodium:buildCMakeRelWithDebInfo[arm64-v8a]'.
[RUN_GRADLEW] > com.android.ide.common.process.ProcessException: ninja: Entering directory `/private/var/folders/zj/kppl1wn13_j6ndc84b1qzb9c0000gn/T/eas-build-local-nodejs/2d00feb0-5c83-4023-8a50-909b1b50b19b/build/node_modules/react-native-libsodium/android/.cxx/RelWithDebInfo/6j1y323g/arm64-v8a'
[RUN_GRADLEW]   [1/4] Building CXX object CMakeFiles/libsodium.dir/cpp-adapter.cpp.o
[RUN_GRADLEW]   FAILED: CMakeFiles/libsodium.dir/cpp-adapter.cpp.o 
`

Any idea what's wrong? Or am I missing something installed on my Mac?

--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.4.1)
 
 set (CMAKE_VERBOSE_MAKEFILE ON)
-set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_CXX_STANDARD 14)
 
 # add directories to "include" search paths
 include_directories(

